packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -72,6 +72,19 @@ class ContractAddresses {
 
 const REGISTRY_ADDRESS = '0x000000000000000000000000000000000000ce10'
 
+const isCoreContract = async (contractName: string) => {
```

Comment:
Would it be better to call this `isProxiedContracts`? Is it the case that there are no (and there won't be) contracts with a corresponding Proxy that wouldn't be considered core Celo contracts? 

Would using `CeloContractName` from `lib/registry-utils.ts` be more appropriate?

Reply:
I did both!